### PR TITLE
feat: change merge rules script

### PR DESCRIPTION
**Pull request by alexanderleegs**
This PR adds a small script meant to be run once, to disable the ability for all isomer repos to squash and merge. The reason for this is to prevent users from accidentally using squash and merge to merge prs, which would cause issues in future when attempting to merge staging to master again.

### CODE REVIEW

**Review by kishore03109 — APPROVED** [review]
approving this code as-is as long as you have tested it on 1 repo and it works cuz it doesnt make sense to optimise something that is only going to be run once. 

However, I am concerned about the newer sites that are being created without this guarantee. Could we copy this this over to our BE and add this as part of the site creation process?

File: update-merge-rules/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  "isomer-conversion-scripts",
+  "isomer-wysiwyg",
+  "isomer-slackbot",
+  "isomer-tooling",
+  "generate-site",
+  "travisci-scripts",
+  "recommender-train",
+  "editor",
+  "ci-test",
+  "infra",
```
Comment (kishore03109) [inline update-merge-rules/index.js:20]: missing isomercms-infra too

Comment (alexanderleegs) [reply update-merge-rules/index.js:20]: Thanks for catching, https://github.com/isomerpages/isomer-tooling/pull/47/commits/77e3b2719af3e9eab5ae1ddc017c0f010cc387b0!

**Comment by alexanderleegs** [comment]
> approving this code as-is as long as you have tested it on 1 repo and it works cuz it doesnt make sense to optimise something that is only going to be run once.
> 
> However, I am concerned about the newer sites that are being created without this guarantee. Could we copy this this over to our BE and add this as part of the site creation process?

Yup, this has been tested on a repo already (a-test-v4) and verified that the functionality is working. It's also relatively easy to reverse in case we miss out any of our admin repos, since that involves toggling the setting on an admin repo which has been configured wrongly, which we shouldn't have too many of. The associated backend pr is already up!

**Pull request merged**